Makefile

```diff
@@ -77,7 +77,7 @@ KERNELPARAMS :=
 ifeq (,$(filter-out centos rhel,$(distro)))
```

Comment:
We have packages for `qemu-cc` too, need to be added to the test, this conditional should be only for rhel IMHO